static/js/biomedical/disease/types.ts

```diff
@@ -34,6 +36,16 @@ export interface DiseaseSymptomAssociationData {
   oddsRatio: number;
 }
 
+// specialization of NamedNode for disease parent data types
+export type DiseaseParent = NamedNode;
+
+export interface DiseaseParentTree {
+  // name of the parent
```

Comment:
and just name of the current node in the tree

Reply:
Done!